French translation

daamien opened the pull request:
Hi ! 

Here's the French translation, it's been proof-read by my colleague Etienne Bersac. If you know french-speaking Red Hat employees that are familiar with the framework, we'd be happy to hear his/her feedback. 

Regards,

ruhbehka commented:
This is awesome, thank you both so much! Working on figuring out how to best merge it into the ODF repo. I'll invite some French Red Hatters to review, and then when everyone is good with it, I can port it over to .odp and .pdf as well... give me a couple days to sort out specifics on how to organize it. :)

brunojean commented on ODF-community.french.md:
```diff
@@ -3,7 +3,9 @@ Cadre de Décicion Ouverte
 
 Version Communitaire 1.0.3
```
communautaire instead of Communitaire

daamien replied:
@brunojean: Applied. Thanks !

EmilienM commented:
:+1: excellente traduction, merci beaucoup! :-)

whit537 commented:
Hey all, coming here from IRC at @ruhbehka's invitation to think about how to bring this PR into the repo—first translation, yay! 🇫🇷  💃

The thing with translations is that they're always at least a _little_ out of date. So the first thing you need to decide is whether to store translations _within_ this repo, and have to deal with the friction here, or adopt a policy of maintaining translations in separate repos, in which case the burden falls more squarely on the translation maintainers' shoulders. If you bring the translation in here then it's a bit more official and you're accepting a bit more of the responsibility for keeping it up to date. So that's one decision.

If you decide to keep translations in separate repos, you can still link to them easily enough from the README here.

If you do decide to bring translations into this repo, then I'd suggest coming up with a directory layout based on [locale codes](https://en.wikipedia.org/wiki/Locale_(computer_software)) such as `en_US`, where the first part is a language code and the second part is a country code. So `en_US` is American English, `fr_FR` would be French French ☺️, `pt_BR` would be Brazilian Portuguese, etc. You'd then file all deliverables under these directories. Here's one possible layout:

```
en_US/ODF-community.md
en_US/ODF-community.odp
en_US/ODF-community.pdf
en_US/Maturity-model-ODF.odt
en_US/Maturity-model-ODF.pdf
fr_FR/ODF-community.md
fr_FR/ODF-community.odp
fr_FR/ODF-community.pdf
fr_FR/Maturity-model-ODF.odt
fr_FR/Maturity-model-ODF.pdf
```

The `en_US/ODF-community.md` document would be the source document (you can decide whether to leave it in the top-level directory or nest it under `en_US`, and/or add another layer of nesting—maybe a `translations` top-level directory with local codes under that?), and every translation should then have a note at the top, "Translated from version 809a986806c32bd7808ee3a9413f10bf136a15fc," using Git's sha-1 hash (available various places in the GitHub UI) or a more human-readable version number ("1.0.1") as appropriate.

Lastly, it might be helpful to have a "translations" [team](https://help.github.com/articles/setting-up-teams/) so you can easily ping all translators when there is new content ready to go, or you're making a push to reach a versioning milestone and would like to sync up translations as part of that.

Hopefully that gives you some ideas about how to proceed here! Good luck! :-)

clopnis commented:
Beau boulot ! 👍

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+**Méritocracie**
+
+Dans un méritocracie, les bonnes idées peuvent venir de n'importe où et la
```
s/un/une/, if I am not wrong.

daamien replied:
done

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Solliciter les usagers internes et les parties prenantes 
+très tôt, en particulier ceux qui pourraient ne pas être d'accord.<br/><br/>
+ • Chercher des perspectives variées et sous-représentées (emplacement
+géographique, ethnie, départements, niveau hiérarchique, genre, age, etc. )<br/><br/>
```
I think that since "ethnie" and all the others are singular, we should keep "département" as singular too.

pilou- replied:
s/age/âge/

daamien replied:
done

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+très tôt, en particulier ceux qui pourraient ne pas être d'accord.<br/><br/>
+ • Chercher des perspectives variées et sous-représentées (emplacement
+géographique, ethnie, départements, niveau hiérarchique, genre, age, etc. )<br/><br/>
+ • Promouvoir la collaboration et fournir des canaux de feedback<br/><br/>
```
What about 'canaux de réponse" rather than "canaux de feedback" ?

daamien replied:
ok

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Quel est l'impact potentiel sur l'organisation ? Sur la culture ?<br/><br/>
+ • Que faut-il inclure dans le planning ?<br/><br/>
+ • Qui est concerné par le problème que nous essayons de régler?<br/><Br/>
+ • A qui devons ou voulons demander de l'aide ?<br/><br/>
```
I think 'nous' is missing somewhere in that sentence.

daamien replied:
done

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Confidentialité, vie privée, et contraintes légales<br/><br/>
+ • Controverses potentielles<br/><br/>
+ • Impact sur la culture Red Hat et les futures décisions<br/><br/>
+ • Roles + responsibilités (modèle OPT model: https://github.com/red-hat-people-team/opt-model/) <br/><br/>
```
"modèle OPT model"  might be replaced by "modèle OPT", IMHO.

pilou- replied:
s/responsibilités/responsabilités/

daamien replied:
done

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+<td rowspan=3 width=33%><strong>Solliciter usagers + collaborateurs</strong><br/>
+ • Rassembler des retours auprès d'usagers internes et de personnes qui vont
+vous aider (questionnaires, entretiens, groupes témoins, etc.)<br/><br/>
+ • Faciliter la participation + les gestion de projet. Demander aux usagers
```
Either 'la gestion', or we need to use the plural form for "gestions". I would favor using "la gestion" since we say "la partitipation".

daamien replied:
agreed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+<strong>Définir les attentes dès le départ</strong><Br/>
+ • Etre spécifique à propos des types de feedback que vous recherchez et à
+propos de qui pris les décisions. <Br/><Br/>
```
shouldn't it be "à propos de qui sont prises les décisions" ?

mscherer replied:
Also, people told me we need to use "Être" rather than "Etre".

daamien replied:
translated into "à propos de qui prend les décisions"

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+<strong>Planifier la transition</strong><br/><br/>
+ • Developper et rassembler le feedback sur la communication, la gestion du
+changement et les plans d'adoption<br/><Br/>
+ • Imaginez comment des personnes ennervées pourraient répondre ( via un memo
```
s/ennervées/énervées/

daamien replied:
done

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+</td>
+<td>
+ • Comment allons-nous prendre les décisions?<Br/><Br/>
+ • Quel usagers internes, parties prenantes et collaborateurs allons-nous impliquer ?<br/><br/>
```
Shouldn't it be "Quels" ?

daamien replied:
Agreed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Comment allons-nous prendre les décisions?<Br/><Br/>
+ • Quel usagers internes, parties prenantes et collaborateurs allons-nous impliquer ?<br/><br/>
+ • Comment allons-nous les solliciter et communiquer avec eux ?<br/><br/>
+ • Quels sont les options open source ?<br/><br/>
```
A friend of mine point that's "Quelles" for "options". I also would use "logiciel libre" rather than "open source".

daamien replied:
The english version says `open source` which remains `open source` in French (you can find `code ouvert` from time to time but `open source` is the common expression)

`Logiciel libre` is the translation for `Free Software`, which is not used at all in the original version 

There's a debate about the differences between the 2 terms but let's not have this converation here :)

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Comment allons-nous les solliciter et communiquer avec eux ?<br/><br/>
+ • Quels sont les options open source ?<br/><br/>
+ • Comment le choix d'une technologie ou d'un format propriétaire va limiter
+nosc choix dans le futur ?<br/><br/>
```
s/nosc/nos/

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+<td>
+ • Impact – qui, quelle fréquence et les imprévus<br/>
+ • Où et comment collaborer<Br/>
+ • Roles + responsibilités (modèle OPT : https://github.com/red-hat-people-team/opt-model/) 
```
s/responsibilités/responsabilités/

daamien replied:
done

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+_page 15 sur 15_ 
+
+**Un ensemble de partiques éprouvées qui :**
```
s/partiques/pratiques/

daamien replied:
done

sanpii commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+<br/>
+
+Phase or activity: Planifier, Rechercher <a name="plan-research"></a>
```
English words here.

daamien replied:
fixed

sanpii commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+<br/>
+
+Phase or activity: Launcer, Deployer, Cloturer <a name="launch"></a>
```
English words here.

sanpii commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Expliquer comment fournir du feedback après le lancement<br/><br/>
+ • Signaler quand vous n'êtes pas complètement satisfait de la décision ou que
+vous savez que d'autres ne le seront pas. <br/><br/>
+ • Parteger votre feuille de route et vos critères pour revenir sur la décision<br/><br/>
```
Typo : « Part**a**ger ».

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+  - Offre un guide cohérent pour les équipes et les leaders à propos des
+    attentes culturelles de Red Hat, équilibrant la transparence et la confidentialité
+  - Améliorer l'implication des collaborateurs, le ratio signal-bruit sur les
+    listes-memo
```
I am not sure we need to translate "memo-list", as that's the name of the mailling list, (more info on http://www.managementexchange.com/story/employee-led-evolution-memo-list-red-hat ). I guess we might need to add more context in the original document.

daamien replied:
Thanks for the context.

Added to the glossary

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+_page 14 sur 15_ 
+
+  - Basé sur des principes practiqués par des communautés open source
```
s/practiqués/pratiqués/

sanpii replied:
Typo: « pra~~c~~tiqués ».

daamien replied:
done

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+  - Basé sur des principes practiqués par des communautés open source
+
+    - Recheches par la Fuqua School of Business à l'université de Duke et Diana
```
s/Recheches/Recherches/

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+  - Basé sur des principes practiqués par des communautés open source
+
+    - Recheches par la Fuqua School of Business à l'université de Duke et Diana
+      Martin (2009 – 2010); ressources communautaire supplémentaires
```
It should be a plural for "communautaire"

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+Pense-bête sur la collaboration, la transparence, la confiance, la méritocracy
+et la connection 
+  - [The Open Source Way handbook](https://www.theopensourceway.org/wiki/Introduction) –  guide pour
+créer et entretenir une communauté de crontributeurs
```
s/crontributeurs/contributeurs/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+Dernière mise à jour de la traduction : 21 Mars 2017 
+
+© 2014-2016 Red Hat et contributeurs | Le Cadre de Décision Ouverte ("Open
```
I guess it's `2014-2017`.

daamien replied:
This one is tricky : the original version says `© 2014-2016` so there's a question wether the translated copyright notice applies to the original version or the translated version. If I translate a copyright notice from 1992, I think I'm not supposed to modify the date...

Maybe we should have two copyright notice : one for the original content and one for the translation... 

I'm leaving it as-is for now...

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+    communauté en ligne)
+  - [Prioritizing by impact](http://developerblog.redhat.com/2014/08/27/5-ux-tips-for-developers/),
+voir le tableau de Máirín Duffy : "5 UX Tips for Developers"
+  - [Opensource.com](http://www.opensource.com/) – Un publication soutenue par
```
s/Un/Une/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+disponible sous licence Creative Commons Attribution-ShareAlike 4.0
+International License ([CC BY-SA
+4.0](https://creativecommons.org/licenses/by-sa/4.0/)). Red Hat et le logo
+Shadowman sont des marques déposées de Red Hat, Inc. et enregistrée dans
```
s/enregistrée/enregistrées/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+* Cette version markdown des diapositives est publiée pour faciliter la
+  collaboration et le suivi des changements.
+* La numérotation des pages correspond aux fichiers LibreOffice (.odp) et PDF
+  dans le dépot.
```
s/dépot/dépôt/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+  * [Phase ou activité: Conceptualiser, Définir, Imaginer](#ideation)
+  * [Phase ou activité: Planifier, Rechercher](#plan-research)
+  * [Phase ou activité: Concevoir, Developper, Tester](#design-develop-test)
+  * [Phase ou activité: Lancer, Deployer, Cloturer](#launch)
```
s/Cloturer/Clôturer/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+  * [Phase ou activité: Planifier, Rechercher](#plan-research)
+  * [Phase ou activité: Concevoir, Developper, Tester](#design-develop-test)
+  * [Phase ou activité: Lancer, Deployer, Cloturer](#launch)
+* [Resources](#resources)
```
s/Resources/Ressources/

daamien replied:
done

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+  * [Phase ou activité: Lancer, Deployer, Cloturer](#launch)
+* [Resources](#resources)
+* [Annexes](#appendix)
+  * [Histoire: D'où vient le Cadre de Décision Ouvert](#history)
```
`Cadre de Décision Ouvert` or `Cadre de Décision Ouverte` ?

daamien replied:
It's `Cadre de Décision Ouverte`  the original version states very clearly that the decision itself is open, not just the framework

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+**Comment l'utiliser ?**
+  - Intégrer par étapes le Cadre de Décision Ouverte dans votre projet ou dans
+    votre processus de prise de décisions.
```
s/décisions/décision/

daamien replied:
done

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+impactés négativement. <Br/><Br/>
+
+<strong>Promouvoir des échanges ouverts</strong><br/>
+ • Evaluer, valider et incorporer le feedback<br/><br/>
```
s/Évaluer/Evaluer/

daamien replied:
done

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+</tr>
+<tr>
+<td>Expliquer qui prend la décision, quels problèmes vous essayez de régler, les
+pré-requis et les contraintes en jeu et le processus que vous aller suivre.</td>
```
s/aller/allez/

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+_page 4 sur 15_ 
+
+**Echanges ouverts**
```
s/Echanges/Échanges/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+<td>Solliciter le retour d'autres personnes et collaborer tout au long du
+processus de décision. Rechercher des points de vue variés, notamment ceux des
+détracteurs potentiels.</td>
+<td>Considérer les personnes comme des usagers ayant dans des besoins
```
s/dans//

daamien replied:
done

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+processus de décision. Rechercher des points de vue variés, notamment ceux des
+détracteurs potentiels.</td>
+<td>Considérer les personnes comme des usagers ayant dans des besoins
+contradictoires et des priorités. Quand une décision aide certains usagers
```
It seems a word is missing after `priorité`.

daamien replied:
Sounds good to me. The original versions says `Think of people as customers with competing needs and priorities.`

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Evaluer, valider et incorporer le feedback<br/><br/>
+ • Mettre en valeur les changements effectués suite au feedback<br/><br/>
+ • Si une suggestion n'est pas possible, expliquer pourquoi<br/><br/>
+ • Publier la progression du projet dans un endroit ouvert <br/><br/>
```
Not sure if I would use "endroit ouvert", maybe "publique" convey the meaning in a clearer way.

daamien replied:
Agreed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+<strong>Imaginer l'échec</strong><br/>
+ • Imaginez que c'est le jour de l'annonce et que des personnes sont surprises
+ou ennervées. Qu'est-ce qui a déclenché cela ?<br/><br/>
```
s/ennervées/énervées/

daamien replied:
done

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+Que vous developpiez un logiciel ou que vous essayiez de résoudre un problème
+dans l'entreprise, l'échange ouvert commence en partageant le "code source" avec
+d'autres personnes. Un échange libre des idées est essentiel pour créer un
+environement où chacun a les moyens d'apprendre et d'utiliser l'information
```
s/environement/environnement/

daamien replied:
fixed in multiple locations

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+Les communautés sont formées autour d'un objectif commun. Elles rassemblent des
+idées diverses et partagent le travail. Une fois réunie, une communauté globale
+peut créer au delà des capacités de n'importe quel individu. Elle multiplie les
+efforts et réparti le travail. Ensemble nous pouvons faire plus.
```
s/réparti/répartit/

daamien replied:
done

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+<td>→<br/>→<br/>→</td>
+<td>
+• Adhésion des usagers<br/> • Adoption plus rapide et plus forte<br/> • Les
+meilleures idées gagnent<br/> • Moins de bugs, de problèmes et de impacts
```
s/de impacts/d'impact/

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Quelle est la fenêtre temporelle raisonable pour l'afinage et les retours
+utilisateurs ?<br/><br/>
+ • Avons-nous sous-estimé un point important ? Comment le prendre en compte ?<br/><br/>
+ • Est-ce que la décision doit être revisée ?<br/><br/>
```
I do not think "revisée" is the right word, shouldn't it be "réévaluée" ?

daamien replied:
both sounds good to me. `révisée` would be applied to a car that needs to be checked once every  2 years, so i liked this image. But ``réévaluée`` is fine.

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+• Adhésion des usagers<br/> • Adoption plus rapide et plus forte<br/> • Les
+meilleures idées gagnent<br/> • Moins de bugs, de problèmes et de impacts
+imprévus <br/> • Plus grand engagement des collaborateurs <br/> • Décisions
+alignées sur le stratégie et la culture
```
- s/le/la/
- an expression may be missing: strategy from whom ?

daamien replied:
Original sentence is ``Decisions aligned to strategy and culture`` so I guess it is implied that we talking about the strategy from Red Hat

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+_page 6 sur 15_ 
+
+  - Je comprend pouquoi la décision a été prise et comment elle s'aligne avec la
```
s/comprend pouquoi/comprends pourquoi/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+On ne peut pas plaire à tout le monde. <a name="cant-please-everyone"></a>
+----------------------------------------------------
+
+**Mais qu'on prend une décision ouverte, les gens se disent...**  
```
s/qu'on/quand on/

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Si nous avons fait des version préliminaires, allons-nous continuer à faire
+des améliorations incrémentales basées sur le feedback ?<br/><br/>
+ • A quel point souhaitons nous faire des révisions sur la base du feedback ?<br/><br/>
+ • Quelle est la fenêtre temporelle raisonable pour l'afinage et les retours
```
s/afinage/affinage/ 

Also, I am a bit surprised to use "affinage" here, as I mostly use it in the context of cheese. But that's a valid meaning.

daamien replied:
AFAIK, the term is used in French IT.

For instance `affiner la compréhension des besoins`

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ouvert<br/><br/>
+ • Publier votre processus de conception<br/><br/>
+
+<strong>Batir une diversité de points de vue + un environement inclusif<strong><br/>
```
s/Batir/Bâtir/

pilou- replied:
s/environement/environnement/

daamien replied:
fixed in multiple locations

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+manière ouverte<br/><br/>
+ • Mettre en avant l'utilisation de ce cadre de décision ouverte <br/><br/>
+ • Dire aux collaborateurs ou trouver des informations détaillées<br/><br/>
+ • Monter comment le feedback a modelé la décision ou le projet<br/><br/>
```
s/Monter/Montrer/

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+</tr>
+<tr>
+<td>
+ • Representation de  différent types d'usagers<br/>
```
s/Representation/Représentation/

pilou- replied:
s/différent/différents/

daamien replied:
fixed

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Est-il possible de faire un prototype ou publier très tôt
+afin de récolter des réactions ?<br/><br/>
+ • Comment allons-nous tester ?<br/><Br/>
+ • Quel usagers internes peuvent nous aider à tester ?<br/><br/>
```
s/Quel/Quels/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+l'énervement chez Red Hat, notamment :<br/><br/>
+ • Les décisions, les politiques ou changement qui impactent les collaborateurs
+, par exemple les rémunérations et le programme de bien-être.<br/><br/>
+ • Les changements dans l'environement de travail des collaborateurs<br/><br/>
```
s/environement/environnement/

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+clarifier et faites-les de manière pro-active.<br/><br/>
+
+<strong>Activer vos ambassadeurs</strong><Br/>
+ • Equiper la communauté pour qu'elle vous aide à clarifier les désinformations
```
s/Equiper/Équiper/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Les décisions, les politiques ou changement qui impactent les collaborateurs
+, par exemple les rémunérations et le programme de bien-être.<br/><br/>
+ • Les changements dans l'environement de travail des collaborateurs<br/><br/>
+ • L'implementation d'une technologie propriétaire<br/><br/>
```
s/implementation/implémentation/

daamien replied:
fixed

mscherer commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+<strong>Imaginer l'échec</strong><br/>
+ • Imaginez que c'est le jour de l'annonce et que des personnes sont surprises
+ou ennervées. Qu'est-ce qui a déclenché cela ?<br/><br/>
+ • Identifier les  changement que vous feriez ou les points que vous pourriez
```
s/changement/changements/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+, par exemple les rémunérations et le programme de bien-être.<br/><br/>
+ • Les changements dans l'environement de travail des collaborateurs<br/><br/>
+ • L'implementation d'une technologie propriétaire<br/><br/>
+ • L'utilisation de formats proprietaires<br/><br/>
```
s/proprietaires/propriétaires/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+vous aider (questionnaires, entretiens, groupes témoins, etc.)<br/><br/>
+ • Faciliter la participation + les gestion de projet. Demander aux usagers
+quels outils de collaboration ils préfèrent utiliser. Prévoir la consolidation
+et le publication du feedback.<br/><br/>
```
s/le publication/la publication/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Etre spécifique à propos des types de feedback que vous recherchez et à
+propos de qui pris les décisions. <Br/><Br/>
+ • Publier le processus de décision et la planification du projet plan avec des
+roles, des dates et des contraintes
```
s/role/rôle/

daamien replied:
fixed

mscherer commented:
On the topic of translation, trying to include the various locales community might help (I did ping the gnome team, for example), and decide which glossary to use is also helpful (as there is disagreement on how to translate some words, if we need to translate them, etc). 

Linking to related documentation, such as https://traduc.org/gnomefr/Typographie can help contributor to know about what rules are followed, and the standard of translation.

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+</tr>
+<tr valign="top">
+<td rowspan=3 width=33%><strong>Batir votre communauté</strong><br/>
+ • Demander aux départments qui parmi leur équipe peut fournir du feedback<Br/><Br/>
```
s/départments/départements/

mscherer replied:
Alos, there is usually more than one team, so I think we should use plural for "leur équipe".

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+<strong>Rassurer les personnes qui expriment des doutes</strong><br/>
+ • Inviter l'équipe du projet et les collaborateurs à remonter des risques ou
+des doutes que vous avez sous-estimés.<br/><br/>
+ • Question: Qu'est-ce qui pourrait empêcher la réussite du projet ? Quels
```
I propose a reformulation: `Quels seront les doutes de l'équipe ?`

daamien replied:
Agreed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+des doutes que vous avez sous-estimés.<br/><br/>
+ • Question: Qu'est-ce qui pourrait empêcher la réussite du projet ? Quels
+doutes l'équipe aura ? Que sommes-nous en train d'oublier ?<br/><br/>
+ • Publier les risques et les limitations découvertes en chemin <br/><br/>
```
I propose a reformulation: `Publier au fur et à mesure les risques et les limitations découvertes`

daamien replied:
Agreed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+</tr>
+<tr valign="top">
+<td rowspan=3 width=33%><strong>Commencer par envisager la fin</strong><br/>
+ • Demontrer l'alignment avec la stratégie, la mission, la culture et les
```
s/Demontrer l'alignment/Démontrer l'alignement/

daamien replied:
Agreed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+<tr valign="top">
+<td rowspan=3 width=33%><strong>Commencer par envisager la fin</strong><br/>
+ • Demontrer l'alignment avec la stratégie, la mission, la culture et les
+veleurs de Red Hat<br/><br/>
```
s/veleurs/valeurs/

daamien replied:
fixed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Souligner les étapes que vous avez suivie pour prendre votre décision de
+manière ouverte<br/><br/>
+ • Mettre en avant l'utilisation de ce cadre de décision ouverte <br/><br/>
+ • Dire aux collaborateurs ou trouver des informations détaillées<br/><br/>
```
s/ou/où/

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+<strong>Ouvert par défaut</strong><br/>
+ • Réaffirmer les contraintes et les pré-requis pertinents <br/><br/>
+ • Partager les problèmes légaux, les rapports, et les
+questions de confidentialité concernés<br/><br/>
```
Either remove `concernés` or add `par la décision`

daamien replied:
Agreed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+ • Communiquer les critères de  succès et publier les métriques pertinentes<br/><br/>
+
+<strong>Contribuer en amont</strong><br/>
+ • Publier vos méthodes, les leçons apprises, les communiqués et les critères
```
I propose some reformulations:
- `les méthodes` instead of `vos méthodes`
- `Publier les archives des méthodes` or `Archiver et publier`

daamien replied:
Agreed with both

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+
+**Un ensemble de partiques éprouvées qui :**
+
+  - Conduit à un meilleur alignement entre les décisions d'entreprise et la
```
Proposal: `Harmoniser les décisions d'entreprise et la stratégie [...]` or 'Obtenir des décisions d'entreprise cohérentes avec la stratégie [...]'

daamien replied:
I suggest `Produit des décisions d'entreprise cohérentes avec la stratégie, [...]`

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+    stratégie, les buts, la culture, les valeurs et la mission de notre société
+  - Démontre “A quoi ressemble les bonnes pratiques" en terme de prise de
+    décision et de communication
+  - Offre un guide cohérent pour les équipes et les leaders à propos des
```
Reformulation: s/à propos des/concernant les/ or `Offre un guide cohérent, pour les équipes, des attentes [...]`

daamien replied:
How about : `Offre aux équipes et aux leaders un guide cohérent des  attentes culturelles de Red Hat`

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+  - [Prioritizing by impact](http://developerblog.redhat.com/2014/08/27/5-ux-tips-for-developers/),
+voir le tableau de Máirín Duffy : "5 UX Tips for Developers"
+  - [Opensource.com](http://www.opensource.com/) – Un publication soutenue par
+    Red Hat focalisée sur comment les principes open source peuvent être
```
- s/entrerpises/entrerprises/
- Reformulation: `Une publication soutenue par Red Hat sur la manière dont les principes open source peuvent être appliqués notamment aux entreprises, à l'éducation, aux gouvernements.`

daamien replied:
Agreed

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+_page 13 of 15_ 
+
+  - [Red Hat Multiplier](https://github.com/red-hat-people-team/red-hat-multiplier) –
+Pense-bête sur la collaboration, la transparence, la confiance, la méritocracy
```
- s/méritocracy/méritocracie/
- s/connection/connexion/

daamien replied:
ok

pilou- commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+utilisateurs ?<br/><br/>
+ • Avons-nous sous-estimé un point important ? Comment le prendre en compte ?<br/><br/>
+ • Est-ce que la décision doit être revisée ?<br/><br/>
+ • Est-ce cette décision ouverte nous a mené au résultat désiré ?<br/><br/>
```
s/Est-ce cette/Est-ce que cette/

ruhbehka commented:
This is great - if you can let me know when you're happy with the translation and add the "translated from" version note that @whit537 suggested, I'll put it into a new directory called fr_FR: 
https://github.com/red-hat-people-team/open-decision-framework/pull/21#issuecomment-289949525

Also feel free to add a contributors note at the end, crediting everyone who contributed to the translation!

daamien commented on ODF-community.french.md:
```diff
@@ -0,0 +1,526 @@
+désengager ?  Qui d'autre devons-nous prendre en compte?
+</td>
+<td rowspan=3>
+Il y a un poignée de problèmes qui provoquent en général de la controverse ou de
```
done

daamien commented:
Hey guys !

Thank you all for the proof-reading. I didn't realized that there's was so many typos and basic errors. This is embarassing  😰 

Anyway thanks for all the quick feedback. This is terrific !

I updated the list of contributors here :
https://github.com/dalibo/open-decision-framework/blob/aff5dfb2abcab0258288e485f80da0d5ec2f6332/TRANSLATION.FRENCH.md#traducteurs

Hope I didn't forget anyone !

@ruhbehka : maybe we should wait for a days to see if someone else wants to proof-read it... If there's no more corrections within the next 7 days we'll consider the translation as "ready for merge". I've added the [WIP] tag in the title to make it clear that the proof-reading is still possible.

ruhbehka commented:
Looks great. Can you create a directory called fr-FR/ and put the two french files in there? Then submit the pull request? That seems to be the easiest way for me to merge it into the main repo and preserve the history.

daamien commented:
done.

I believe this is ready for merge. Let me know if you need anything
